tests/runtime/test_grpc_server.py

```diff
@@ -953,6 +956,103 @@ def test_runtime_status_ok_response(runtime_grpc_server):
     assert actual_response.numericRuntimeVersion == 0
 
 
+def test_runtime_info_ok_response(runtime_grpc_server):
+    runtime_info_service: ServicePackage = ServicePackageFactory().get_service_package(
+        ServicePackageFactory.ServiceType.RUNTIME_INFO,
+    )
+
+    runtime_info_stub = runtime_info_service.stub_class(
+        runtime_grpc_server.make_local_channel()
+    )
+
+    runtime_request = RuntimeInfoRequest()
+    runtime_info_response: RuntimeInfoResponse = RuntimeInfoResponse.from_proto(
+        runtime_info_stub.GetRuntimeInfo(runtime_request.to_proto())
```

Comment:
This test passes but how does one test this locally? Wwhen I startup the server for samplelib, I see the `/runtimeInfo` endpoint in REST (see image) and I see protos files `caikit_data_model.runtime.runtimeinforesponse.proto` and `caikit_data_model.runtime.runtimerequest.proto` but I don't see `GetRuntimeInfo` within protos

<img width="1304" alt="Screenshot 2023-10-24 at 6 15 24 PM" src="https://github.com/caikit/caikit/assets/16713314/1bd4c238-b46f-41b2-bff0-be375df56a02">

Reply:
Is the issue that the `service`/`rpc` is not getting generated with `dump_services`? If this test is passing I think it means that the servicer is getting bound correctly. Do you see it with reflection using `grpcui`?

Reply:
> Is the issue that the service/rpc is not getting generated with dump_services
Yes I believe so, there is only `samplelibservice.proto` and `samplelibtrainingservice.proto` but you're right that with `grpcui` it does appear. Should there be a proto file for samplelibruntimeinfoservice?

<img width="578" alt="Screenshot 2023-10-25 at 2 28 14 PM" src="https://github.com/caikit/caikit/assets/16713314/f5d6a79b-f708-4555-a192-ced72cab0036">

Reply:
As defined I don't think the service would have `sample_lib` prepended to the name- it should just be named `runtimeinfoservice.proto`. That's probably what we want- I'd imagine this service definition should stay constant no matter which libraries you're serving in the runtime

Reply:
that makes sense but there isn't a `runtimeinfoservice.proto`...
```
$ ls protos | grep service
samplelibservice.proto
samplelibtrainingservice.proto
```

Reply:
I suspect this is because of how [dump_grpc_services](https://github.com/caikit/caikit/blob/main/caikit/runtime/dump_services.py#L32) is implemented. I think you need to add this new service there!

Reply:
ah-ha that was it! Now the protos are `info.proto`, `caikit_data_model.common.runtime.runtimeinforequest.proto`, and `caikit_data_model.common.runtime.runtimeinforesponse.proto`. 

With info.proto looking like:
```
service Info {
  rpc GetRuntimeInfo(caikit_data_model.common.runtime.RuntimeInfoRequest) returns (caikit_data_model.common.runtime.RuntimeInfoResponse);
}
```

Reply:
Yep, that would be the place to change it